submissions/Barasii/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML-CSS-Popup</title>
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+    <header class="header">
+        <nav class="navigation">
+            <ul class="navigation__menu">
+                <li class="navigation__item">
+                    <a href="#" alt="Email" class="navigation__link">Email</a>   
+                </li>
+                <li class="navigation__item">
+                    <a href="#" alt="Images" class="navigation__link">Images</a>
```

Comment:
`a` tag doesn't have an `alt` attribute. More info in [spec](https://www.w3.org/TR/html50/text-level-semantics.html#the-a-element) (see content attributes list).